src/libs/actions/connections/NetSuiteCommands.ts

```diff
@@ -402,6 +402,118 @@ function updateNetSuiteImportMapping<TMappingName extends keyof Connections['net
+                                        },
+                                    },
+                                    errorFields: {
+                                        customers: null,
+                                        jobs: null,
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+        ],
+        successData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    connections: {
+                        netsuite: {
+                            options: {
+                                config: {
+                                    syncOptions: {
+                                        mapping: {
+                                            customers: mappingValue.customersMapping,
+                                            jobs: mappingValue.jobsMapping,
+                                            pendingFields: {
+                                                customers: null,
+                                                jobs: null,
+                                            },
+                                        },
+                                    },
+                                    errorFields: {
+                                        customers: null,
+                                        jobs: null,
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+        ],
+        failureData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    connections: {
+                        netsuite: {
+                            options: {
+                                config: {
+                                    syncOptions: {
+                                        mapping: {
+                                            customers: oldMappingValue.customersMapping,
+                                            jobs: oldMappingValue.jobsMapping,
+                                            pendingFields: {
+                                                customers: null,
+                                                jobs: null,
+                                            },
+                                        },
+                                    },
+                                    errorFields: {
+                                        customers: ErrorUtils.getMicroSecondOnyxErrorWithTranslationKey('common.genericErrorMessage'),
+                                        jobs: ErrorUtils.getMicroSecondOnyxErrorWithTranslationKey('common.genericErrorMessage'),
```

Comment:
```suggestion
                                        customers: ErrorUtils.getMicroSecondOnyxErrorWithTranslationKey('common.genericErrorMessage'),
                                        jobs: ErrorUtils.getMicroSecondOnyxErrorWithTranslationKey('common.genericErrorMessage'),
```

The API should return an error message for invalid inputs, etc. Can we not use that?

Reply:
Well we have a dropdown for selection and typescript, do you think we need to worry about having the invalid input errors?

Reply:
Perhaps not. Yeah let's consider it a NAB.